nuxt-frontend/pages/portfolio/[id].vue

```diff
@@ -133,3 +133,4 @@ watch(
   }
 }
 </style>
+~/json-data/luxeradio-data.js~/json-data/justly-data.js~/json-data/togness-data.js
```

Comment:
Can you explain this?

Reply:
Maybe it's mistakenly added while squashing. Removed it